### PR TITLE
[DEV-219992] Chrome disabled message

### PR DESCRIPTION
**Pull request by andresguedes**
- Adding message to user showing that Google Chrome is required in case of it is disabled or uninstalled

### CODE REVIEW

**Review by marcosrivereto — APPROVED** [review]

File: trustly-android-sdk/.DS_Store
Comment (renatodelpupo) [inline trustly-android-sdk/.DS_Store:1]: If possible, I would suggest to not commit this file.

**Review by renatodelpupo — APPROVED** [review]

File: trustly-android-sdk/.DS_Store
Comment (lukevance) [reply trustly-android-sdk/.DS_Store:1]: Also, can we just move all the files up one directory? It seems like the SDK is actually buried one layer deep, is this necessary?

Comment (andresguedes) [reply trustly-android-sdk/.DS_Store:1]: This files will be removed in this [PR](https://github.com/TrustlyInc/trustly-android/pull/15)

**Comment by rhaissatrustly** [comment]
QA Validation - Approved

**Pull request merged**